hyperspy/misc/utils.py

```diff
@@ -907,102 +906,97 @@ def stack(signal_list, axis=None, new_axis_name='stack_element',
     from hyperspy.signals import BaseSignal
     import dask.array as da
     from numbers import Number
-    # TODO: remove next time
```

Comment:
I think this need to stay for now and it should be removed in v2.0 because it has been deprecated in v1.2 (see https://github.com/hyperspy/hyperspy/commit/28be16949b183585c8d4ff85d4d72762ce8c6a15 in #1219). The comment could be more explicit!

Reply:
Done.